pycontw/pycontw-frontend
# feat(staff page)-switch layout with ccip

## Description
Add a custom layout "ccip" to hide footer and header at page staff, switch the layout based on CCIP value

## Steps to Test This Pull Request
1. Type “<localhost>/2023/en-us/about/staff?ccip=1” in url
2. Scroll to the top see if header is hidden
3. Scroll down the bottom see if footer is hidden

## Expected behavior

When ccip is true, footer and header are hidden as picture:
<img width="207" alt="image" src="https://github.com/pycontw/pycontw-frontend/assets/26858727/c0fc4c23-59c2-4008-a3fd-65d25197a476">

If false:
<img width="203" alt="image" src="https://github.com/pycontw/pycontw-frontend/assets/26858727/141fbc56-febc-4618-87c1-9c6a1e2bed6c">

## Related Issue
#413
